### ISSUE TITLE
Need new release for 64-bit version of Rocksmith

### ISSUE THREAD

**Issue by JustinAiken** [rscustom/RSInjector-Mac#6]
64-bit branch just came out of beta; as expected this patch no longer works...

**Comment by JustinAiken** [comment]
I tried compiling the project as is, but set for 64 bit architecture and a later macOS target - no luck crashes on startup. 

```
Thread 0 Crashed:: MainThrd  Dispatch queue: com.apple.main-thread
0   libRSBypass.dylib                 0x0000000111828831 bCompare + 49
1   libRSBypass.dylib                 0x00000001118288e4 FindPattern + 68
2   libRSBypass.dylib                 0x000000011182896f +[RSBypass load] + 63
3   libobjc.A.dylib                   0x00007fff6e91b606 load_images + 1324
4   dyld                              0x000000011a77f19f dyld::notifySingle(dyld_image_states, ImageLoader const, ImageLoader::InitializerTimingList) + 418
```

I would have been really surprised if it did work 😛

**Comment by buylife** [comment]
Hey guys where I can get precompiled RSBypass/RSInjector 64 bit? Thank you!

**Comment by JustinAiken** [comment]
This was fixed in 9dec806eb92eada0604b94b42713a82e5df10cff
